backend/apps/api/decorators/cache.py

```diff
@@ -36,6 +36,8 @@ def cache_response(
     def decorator(view_func):
         @wraps(view_func)
         def _wrapper(request, *args, **kwargs):
+            if settings.IS_E2E_ENVIRONMENT:
```

Comment:
Why e2e needs this change?

Reply:
Yes, let's add cache service for e2e instead. The closer to production architecture the better -- for both local and CI/CD cases.